spec/onfido_spec.rb

```diff
@@ -35,6 +35,27 @@
       end
     end
 
+    describe 'using a US API token' do
```

Comment:
Can we add a spec for what happens if it's not an API key with a "region bit"?

Reply:
There's a `describe 'using an old API token'` below to test using the old-style API keys (no region bit), is that what you mean? As per the other comments, I'm finishing some related updates and will be pushing them in a bit.

Reply:
Ah, I didn't spot that test.